database/postgres_store.go

```diff
@@ -0,0 +1,95 @@
+package database
+
+import (
+	"context"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/jackc/pgx/v4"
```

Comment:
please use https://github.com/jmoiron/sqlx/ (sqlx is used in all our other projects too)